src/ResourceManager/IotHub/Commands.IotHub/IotHub/RemoveAzureRmIotHubCertificate.cs

```diff
@@ -0,0 +1,101 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
Please implement PassThru here: https://github.com/Azure/azure-powershell/blob/342cf641dd72554c9bd601bb0553eb879b99c6c9/src/ResourceManager/Profile/Commands.Profile/Default/ClearAzureRmDefault.cs#L40.  When the user specifies PassThru, they will get back either "True" if the operation succeed, or "False" if it did not.

Reply:
Done